### DISCUSSION TITLE
Deployment to preview are not happening when I push code to develpment branch in github

### DISCUSSION THREAD

**Discussion by tp-auto-tech** [vercel/community#7294] [Help]
### Summary

Hey guys, I wish you are all fantastic!

I'm facing an issue here. I recently changed the github repo from a personal account repo to an organization one and since new push to the development branch is not triggering a new deployment into Vercel preview.
Moreover, I cannot seem to find where I can connect the development Git branch to Vercel Preview.

Please help! 😄 
Cheers,
TP

### Steps to Reproduce

I'm not sure how this can be replicated, but:
1. Navigate to the project Settings -> Git
2. There is no place to connect a branch to Preview

![image](https://github.com/vercel/community/assets/127350870/7548d211-6cf3-4386-9e4d-229b06151b25)

**Comment by versecafe** [comment]
@tp-auto-tech Vercel creates a preview deployment for every commit you make regardless of branch.

[a comment by an automated account is not shown]
